GsaGH/Parameters/5_Results/GsaResult.cs

```diff
@@ -931,17 +932,17 @@ public class GsaResult {
       }
 
       if (Type == CaseType.AnalysisCase) {
-        if (!ACaseNodeDisplacementValues.ContainsKey(nodelist)) {
+        if (!ACaseNodeDisplacementValues.ACaseResultValues.ContainsKey(nodelist)) {
           if (!ACaseNodeResults.ContainsKey(nodelist)) {
             ACaseNodeResults.Add(nodelist, AnalysisCaseResult.NodeResults(nodelist));
           }
 
-          ACaseNodeDisplacementValues.Add(nodelist,
+          ACaseNodeDisplacementValues.AddACaseValue(nodelist,
             ResultHelper.GetNodeResultValues(ACaseNodeResults[nodelist], lengthUnit));
         }
 
         return new Tuple<List<GsaResultsValues>, List<int>>(new List<GsaResultsValues> {
-          ACaseNodeDisplacementValues[nodelist],
+          ACaseNodeDisplacementValues.ACaseResultValues[nodelist],
         }, Model.Model.Nodes(nodelist).Keys.ToList());
       }
```

Comment:
Should we move this to a method in the new class?

Reply:
not a 1 step I suppose